docs/dev/arch_decision.md

```diff
@@ -0,0 +1,7 @@
+# Architecture Decision Records
+
+The intention is to document deviations from a standard Model View Controller (MVC) design.
+
+!!! warning "Developer Note - Remove Me!"
```

Comment:
Fill or remove

Reply:
@DistantVoyager This is up to package maintainers pls.

Reply:
Not needed. Will update PR to remove `arch_decision.md`